src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUY_OPERATION = "buy";
+    private static final String DATA_SEPARATOR = ",";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
+        int buySum = 0;
+        String line;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            while ((line = reader.readLine()) != null) {
+                String[] data = line.split(DATA_SEPARATOR);
+                if (data[0].equals(SUPPLY_OPERATION)) {
+                    supplySum += Integer.parseInt(data[1]);
+                } else if (data[0].equals(BUY_OPERATION)) {
+                    buySum += Integer.parseInt(data[1]);
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + fromFileName, e);
+        }
```

Comment:
it's not ok, you should create two separate methods - one for reading from file and the second one for processing of read lines from file